cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java

```diff
@@ -160,33 +163,30 @@ private PublicationContext buildPublicationContext()
         return null;
     }
 
-    private Book buildBookForPublicationContext() throws InvalidIsbnException {
+    private Book buildBookForPublicationContext() throws InvalidIsbnException, InvalidUnconfirmedSeriesException {
         List<String> isbnList = extractIsbn().stream().collect(Collectors.toList());
-        return new Book.Builder()
-                .withIsbnList(isbnList)
-                .withPublisher(extractPublisherName())
-                .build();
+        return new Book(null, null, null, buildUnconfirmedPublisher(), isbnList);
```

Comment:
replace nulls with constants,EMPTY_SERIES,EMPTY_SERIES_TITLE,EMPTY_SERIES_NUMBER.
 A simpler constructor would be easier to handle

Reply:
Bump the model to 0.13.1 and use the simple constructor (the one without unconfirmedSeriesTitle)